src/lib/modules/console/index.ts

```diff
@@ -6,12 +6,12 @@ const EmbarkJS = require("embarkjs");
 const IpfsApi = require("ipfs-api");
 const stringify = require("json-stringify-safe");
 import Web3 from "web3";
-import { Embark, Events } from "../../../typings/embark";
+import { Embark, EventEmitter } from "../../../typings/embark";
```

Comment:
As mentioned above, we can import `EventEmitter` from `lib/core/events` as we already define a type there.

Reply:
Sadly no, I tried doing that but because Embark has not been move to be a type, the type check do not allow me to make that change